### PR TITLE
Adding additional French translations

### PR DESCRIPTION
**Pull request by sylviamclaughlin**
# Summary | Résumé

Adding additional translations to make the app fully bilingual.

### CODE REVIEW

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -25,22 +25,47 @@ msgstr " Approuver"
...
+msgid "Manager approved the request"
+msgstr "Le responsable a approuvé la demande"
+
+#: approve/forms.py:93 approve/forms.py:163 submit_request/forms.py:119
+msgid "Manager denied the request"
+msgstr "Le directeur a refusé la demande"
+
+#: approve/forms.py:164
+msgid "Date sent to S32 approver"
+msgstr "Date d'envoi à l'approbateur S32"
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:59]: I'm not 100% sure what's the French word for this? I think it's the right one though :)

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -25,22 +25,47 @@ msgstr " Approuver"
...
+
+#: approve/forms.py:165
+msgid "Date S32 approver reviewed the request"
+msgstr "Date à laquelle le responsable a examiné la demande"
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:63]: respsable -> approbateur S32

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -25,22 +25,47 @@ msgstr " Approuver"
...
+
+#: approve/forms.py:166
+msgid "S32 approver approved the request"
+msgstr "Le responsable a approuvé la demande"
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:67]: le responsable -> l'approbateur S32

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -143,49 +168,96 @@ msgstr "Administrateur de sauvegarde"
...
+#: internal_ops/forms.py:42 internal_ops/forms.py:172 internal_ops/forms.py:273
+#: internal_ops/forms.py:375
+msgid "Date Manager reviewed the request"
+msgstr "Date à laquelle le responsable a examiné la demande"
 
-#: internal_ops/forms.py:48 internal_ops/forms.py:181
+#: internal_ops/forms.py:43 internal_ops/forms.py:167 internal_ops/forms.py:274
+#: internal_ops/forms.py:376
+msgid "Manager Approved"
+msgstr "Gestionnaire approuvé"
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:179]: I think we could use "Approuvé par le gestionnaire"

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -143,49 +168,96 @@ msgstr "Administrateur de sauvegarde"
...
+
+#: internal_ops/forms.py:44 internal_ops/forms.py:168 internal_ops/forms.py:275
+#: internal_ops/forms.py:377
+msgid "Manager Denied"
+msgstr "Gestionnaire refusé"
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:184]: If we change the one before, we could say "Refusé par le gestionnaire"

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -143,49 +168,96 @@ msgstr "Administrateur de sauvegarde"
...
+
+#: internal_ops/forms.py:45 internal_ops/forms.py:173
 msgid "Submitted By"
 msgstr "Proposé par"
 
-#: internal_ops/forms.py:49 internal_ops/forms.py:182
+#: internal_ops/forms.py:46 internal_ops/forms.py:169 internal_ops/forms.py:174
+#: internal_ops/forms.py:276 internal_ops/forms.py:378
 msgid "Date Info Requested"
 msgstr "Informations sur la date demandées"
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:193]: I think the French syntax might be something like:
"Date à laquelle des informations ont été demandées"

That said, I haven't seen this string in the UI

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -143,49 +168,96 @@ msgstr "Administrateur de sauvegarde"
...
+#: internal_ops/forms.py:47 internal_ops/forms.py:170 internal_ops/forms.py:175
+#: internal_ops/forms.py:277 internal_ops/forms.py:379
 msgid "Info Requested"
 msgstr "Informations demandées"
 
-#: internal_ops/forms.py:56 internal_ops/forms.py:184
+#: internal_ops/forms.py:48 internal_ops/forms.py:176 internal_ops/forms.py:279
+#: internal_ops/forms.py:381
 msgid "Fund Center"
 msgstr "Centre de fonds"
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:203]: I'd have to check with official documentation as I'm not sure it's the same term used in French

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -143,49 +168,96 @@ msgstr "Administrateur de sauvegarde"
...
 
-#: internal_ops/forms.py:367
+#: internal_ops/forms.py:171 internal_ops/forms.py:278
+#: internal_ops/forms.py:380
+msgid "Date Sent to S32 Approver"
+msgstr "Date d'envoi à l'approbateur S32"
+
+#: internal_ops/forms.py:281 internal_ops/forms.py:383
+msgid "S32 Approver approved"
+msgstr "Approbateur S32 approuvé"
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:234]: I think the syntax might be: "Approuvé par l'approbateur S32"

Although that does sound a bit odd...

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -206,76 +278,76 @@ msgstr ""
...
 msgstr "Nous avons réussi à envoyer un e-mail à l'approbateur S32."
 
-#: internal_ops/views.py:256
+#: internal_ops/views.py:255
 msgid ""
 "There was an error sending the notification emails to the S32 approver or "
 "requestor."
 msgstr ""
 "Une erreur s'est produite lors de l'envoi des e-mails de notification à "
 "l'approbateur ou au demandeur S32"
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:322]: I think this last part probably is more along those lines:
"l'approbateur S32 ou au demandeur"

**Review by gcharest — CHANGES_REQUESTED** [review]
I reviewed the strings and while I'm comfortable with some of the proposed changes, I'd probably request support from the translation services team 😄 

I'm also not sure how we can change the terms in a way that uses inclusive writing.

**Comment by sylviamclaughlin** [comment]
@gcharest - I have made all the changes! Thank you for reviewing the translations

**Review by gcharest — APPROVED** [review]
Approved! I'd recommend another review by Translation Services whenever possible for a better consistency with the terms used 😄

**Pull request merged**